r55/Cargo.toml

```diff
@@ -6,14 +6,18 @@ license.workspace = true
 repository.workspace = true
 
 [dependencies]
-#revm = { git = "https://github.com/r0qs/revm" }
 eth-riscv-interpreter.workspace = true
 eth-riscv-syscalls.workspace = true
 
-revm = "9.0.0"
+#revm = { git = "https://github.com/r0qs/revm" }
```

Comment:
we can remove this actually

Reply:
@0xurb can you just remove this? then we can merge right away

Reply:
@leonardoalt remove `r0qs` revm? Can you clarify - we keeping this
```toml
revm = { version = "9.0.0", features = ["std"] }
```
?

Reply:
ah yes I meant removing the old commented out revm fork

Reply:
removed